src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -205,9 +324,19 @@ public void close() throws IOException {
 
     @Override
     public String toString() {
-        return new StringBuilder("build ").append(tag != null ? "-t " + tag + " " : "")
-                .append(noCache ? "--nocache=true " : "").append(quiet ? "--quiet=true " : "")
-                .append(!remove ? "--rm=false " : "").toString();
+        return new StringBuilder("build ")
+                .append(tag != null ? "-t " + tag + " " : "")
+                .append(remote != null ? "--remote=" + remote.toString() + " " : "")
+                .append((quiet != null && quiet) ? "--quiet=true " : "")
+                .append((noCache != null && noCache) ? "--nocache=true " : "")
+                .append((pull != null && pull) ? "--pull=true" : "")
+                .append((remove != null && !remove) ? "--rm=false " : "")
+                .append((forcerm != null && forcerm) ? "--forcerm=true " : " ")
+                .append(memory != null ? "--memory=" + memory + " " : "")
+                .append(memswap != null ? "--memswap=" + memswap + " " : "")
+                .append(cpushares != null ? "--cpushares=" + cpushares + " " : "")
+                .append(cpusetcpus != null ? "--cpusetcpus=" + cpusetcpus + " " : "")
+                .toString();
```

Comment:
I doubt that CLI analogy is really needed in the string representations. What should it be for?